packages/page-calendar/src/DayItem.tsx

```diff
@@ -190,6 +196,17 @@ export default React.memo(styled(DayItem)`
     }
   }
 
+  .exportCal-allEvents {
+    right: 3.5rem;
+  }
+
+  .itemDate {
+    padding: 0 0.375rem;
+    border-radius: 0.25rem;
+    color: #000;
```

Comment:
What does this do in dark mode?

Reply:
There is a dark mode? Didn't know that. I'll try to understand what would be best and push a fix on that. Thank you :)

Reply:
And now that I asked that, seems we had a regression in the calandar recently-ish. (When dark mode went it it did have some fixes to make it usable, but atm it really looks bad)

Reply:
... it looks horrible atm. It was completely broken, then I fixed it shortly afterwards, now it seems completely broken again ...

Settings -> theme

Reply:
EDIT: Have a simple fix here to align it with table. https://github.com/polkadot-js/apps/pull/4268

Reply:
Will take care of it - no worries :)